feat: add Review entity and repository

moominIsCute opened the pull request:
## #️⃣ Issue Number<!--- ex) #이슈번호, #이슈번호 -->

- Closes #7

## 📝 요약(Summary)<!--- 변경 사항 및 관련 이슈에 대해 간단하게 작성해주세요. 어떻게보다 무엇을 왜 수정했는지 설명해주세요. -->
-리뷰 엔티티 구현
-리뷰 repositoty 구현

## 🛠️ PR 유형
어떤 변경 사항이 있나요?
- [x] 새로운 기능 추가
- [ ] 버그 수정
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [ ] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## ✅ PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.
- [x] 커밋 메시지 컨벤션에 맞게 작성

moominIsCute committed:
feat(review): add Review entity

- Review 엔티티 생성
- User와 ManyToOne 관계 설정
- equipmentScore, userScore 필드 추가
- content 필드 추가 (최대 1000자)
- BaseEntity 상속으로 createdAt, updatedAt 자동 관리

moominIsCute committed:
feat(review): add ReviewRepository

- JpaRepository 상속으로 기본 Repository 생성

moominIsCute commented on src/main/resources/application.yml:
```diff
@@ -1,3 +1,20 @@
+  datasource:
+    url: jdbc:mysql://localhost:3306/golfRental?useSSL=false&characterEncoding=UTF-8&serverTimezone=Asia/Seoul
+    username: ${SPRING_DATASOURCE_MYSQL_USERNAME:root}
+    password: ${SPRING_DATASOURCE_MYSQL_PASSWORD:root}
```
오 이런 수정합니다.

moominIsCute commented on src/main/resources/application.yml:
```diff
@@ -1,3 +1,20 @@
+
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
```
수정했습니다.

zerone1202 reviewed (approved):
Assignees, Labels 등 추가 부탁드립니다!

AllSungho reviewed (approved):
수정사항 확인했습니다.
고생 하셨습니다!!

moominIsCute commented:
> Assignees, Labels 등 추가 부탁드립니다!

수정했는데 이렇게 하는것 맞을까요?

moominIsCute committed:
refactor(review): add validation and improve security

- Review 엔티티에 @Min, @Max 검증 추가 (점수 1~5 제한)
- spring-boot-starter-validation 의존성 추가
- application.yml 비밀번호 기본값 제거 (보안 강화)
